Userland/Libraries/LibSymbolication/Symbolication.cpp

```diff
@@ -43,13 +43,7 @@ Optional<FlatPtr> kernel_base()
             return {};
         }
         auto kernel_base_str = DeprecatedString { file.value()->read_all(), NoChomp };
-#if ARCH(I386)
-        using AddressType = u32;
-#elif ARCH(X86_64) || ARCH(AARCH64)
```

Comment:
oh! there it is! 😅

(i.e. is it advisable to mark this here as `#if ARCH(X86_64) || ARCH(AARCH64)` and keep unknown architecture error for other cases? maybe flag should be just different?)